### PR TITLE
fix: fix broken link and use improved reference to Google Docs

### PR DESCRIPTION
**Pull request by machielvdw**
Closes #ISSUE_NUMBER

## Checklist

- [x] Pull request is made to the "develop" branch
- [ ] Tests added
- [x] Documentation added/updated
- [x] Code has been formatted with Prettier
- [x] Edits from maintainers are allowed

### CODE REVIEW

File: packages/docs/templates/footer.yaml.njk [hunk 1]
```diff
@@ -249,7 +249,7 @@
 
 - id: footer_divider
   type: Divider
-- _ref: templates/built_with_lowdefy.yaml
+{# - _ref: templates/built_with_lowdefy.yaml #}
```
Comment (SamTolmay) [inline packages/docs/templates/footer.yaml.njk:252]: Please add this back, the other footer will be removed once we add a license key

File: .changeset/ten-actors-mate.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'@lowdefy/docs': patch
+---
+
+fix broken link and use improved reference to Google Docs
```
Comment (SamTolmay) [inline .changeset/ten-actors-mate.md:5]: `Fix broken link and use improved reference to Google Docs.`

File: packages/docs/templates/footer.yaml.njk (hunk 1 above)
Comment (machielvdw) [reply packages/docs/templates/footer.yaml.njk]: Done - 9f29bb0645b59eb9a2958e6e3e1aaefb9ba0cea9

File: .changeset/ten-actors-mate.md (hunk 2 above)
Comment (machielvdw) [reply .changeset/ten-actors-mate.md]: Done - 9f29bb0645b59eb9a2958e6e3e1aaefb9ba0cea9

**Review by SamTolmay — APPROVED** [review]

**Pull request merged**